the-blue-alliance-ios/View Elements/Match/Breakdown/MatchBreakdownConfigurator2020.swift

```diff
@@ -0,0 +1,258 @@
+import Foundation
+import UIKit
+
+private class BreakdownStyle2020 {
+    public static let bottomImage = UIImage(systemName: "rectangle")
+    public static let outerImage = UIImage(systemName: "hexagon")
+    public static let innerImage = UIImage(systemName: "circle")
+    public static let checkImage = UIImage(systemName: "checkmark")
```

Comment:
We shouldn't need to include either `checkImage` or `xImage` since we can pull them both from the shared `BreakdownStyle`